app/src/main/java/com/codesoom/assignment/core/application/MemberService.java

```diff
@@ -0,0 +1,68 @@
+package com.codesoom.assignment.core.application;
+
+import com.codesoom.assignment.core.domain.Member;
+import com.codesoom.assignment.core.domain.MemberRepository;
+import com.codesoom.assignment.web.dto.MemberData;
+import com.codesoom.assignment.web.exception.MemberNotFoundException;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+
+@Service
+@Transactional
+public class MemberService {
+
+    private final MemberRepository memberRepository;
+
+    public MemberService(MemberRepository memberRepository) {
+        this.memberRepository = memberRepository;
+    }
+
+    /**
+     * 신규 회원를 등록합니다.
+     * @param  memberData - 신규 등록할 회원 정보(이름, 비밀번호, 이메일 등).
+     * @return member - 고유 ID를 발급받은 신규 회원.
+     */
+    public Member saveMember(MemberData memberData) {
+        Member member = Member.builder()
+                .name(memberData.getName())
+                .password(memberData.getPassword())
+                .email(memberData.getEmail())
+                .build();
+
+        return memberRepository.save(member);
+    }
+
+    /**
+     * 기존 회원를 삭제합니다.
+     * @param  id - 삭제할 특정 회원의 고유 ID.
+     * @throws MemberNotFoundException - ID에 해당하는 회원이 없다면, 회원 정보를 찾을 수 없다는 에러 발생.
+     */
```

Comment:
`기존`이라는 단어는 아예 안 쓰는 것이 좋습니다. 이 메소드를 호출하는 시점에서 `기존`회원이 아닌 회원도 있을까요? 있다 하더라도 `기존`회원인지 `기존`회원이 아닌지가 이 메소드에서 중요할까요? 

```suggestion
    /**
     * 회원 정보를 찾아 삭제합니다.
     * @param  id 삭제할 회원의 식별자
     * @throws MemberNotFoundException 회원을 찾을 수 없는 경우
     */
```

`@throws`는 어떠한 경우인지를 설명해주는 것으로 충분합니다. `~한다면 회원 정보를 찾을 수 없다는 에러 발생`은 필요 이상으로 설명하고 있어요.

Reply:
당연한 것을 기입할 필요는 없겠네요 ㅎㅎㅎ..